specification/securityinsights/resource-manager/Microsoft.SecurityInsights/stable/2023-11-01/SourceControls.json

```diff
@@ -0,0 +1,1005 @@
+            "description": "Updated",
+            "schema": {
+              "$ref": "#/definitions/SourceControl"
+            }
+          },
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/SourceControl"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/2.0/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/sourcecontrols/{sourceControlId}/delete": {
+      "post": {
+        "x-ms-examples": {
+          "Delete a source control.": {
+            "$ref": "./examples/sourcecontrols/DeleteSourceControl.json"
+          }
+        },
+        "tags": [
+          "SourceControls"
+        ],
+        "description": "Delete a source control.",
+        "operationId": "SourceControls_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../common/2.0/types.json#/parameters/WorkspaceName"
+          },
+          {
+            "$ref": "#/parameters/SourceControlIdParameter"
+          },
+          {
+            "$ref": "#/parameters/RepositoryAccess"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Source control deleted.",
+            "schema": {
+              "$ref": "#/definitions/Warning"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/2.0/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "RepoList": {
+      "description": "List all the source controls.",
+      "properties": {
+        "nextLink": {
+          "description": "URL to fetch the next set of repositories.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "value": {
+          "description": "Array of repositories.",
+          "items": {
+            "$ref": "#/definitions/Repo"
+          },
+          "type": "array",
+          "x-ms-identifiers": []
+        }
+      },
+      "required": [
+        "value"
+      ],
+      "type": "object"
+    },
+    "Repo": {
```

Comment:
Since repo is a resource, being PUT and Get via ARM APIs, it should be a proxy ARM resource.

Reply:
This is not a real recourse. It is a definition of external Git repository connected to the object and is a property of `SourceControl`.